{ 'pack_motion_2': { 'head': [ { 'duration': 2.984,
                                 'name': u'pack_final',
                                 'positions': [1.4452, -0.0],
                                 'starttime': 2.029}],
                     'left_arm': [ { 'duration': 1.96,
                                     'name': u'pack_final',
                                     'positions': [ -3.1201,
                                                    0.0002,
                                                    0.0001,
                                                    -0.0002,
                                                    0.9425,
                                                    0.0,
                                                    0.0],
                                     'starttime': 0.0}],
                     'left_leg': [],
                     'right_arm': [ { 'duration': 1.96,
                                      'name': u'pack_final',
                                      'positions': [ 3.1201,
                                                     -0.0002,
                                                     -0.0001,
                                                     0.0002,
                                                     -0.9425,
                                                     -0.0,
                                                     -0.0],
                                      'starttime': 0.0}],
                     'right_leg': [],
                     'torso': [ { 'duration': 2.984,
                                  'name': u'pack_final',
                                  'positions': [-0.0, 1.5728],
                                  'starttime': 2.013}]}}
